### DISCUSSION TITLE
editor.on('init') not firing

### DISCUSSION THREAD

**Discussion by connorhvnsen** [ueberdosis/tiptap#774] [General Questions]
In my component, the event listener for the editor initialization is never fired when called inside of `mounted` or `created`. I haven't opened this as a bug yet as I'm not sure this is the proper way to register this listener. Any ideas? Thanks!
```
mounted: function() {
        console.log(this.editor)

        this.editor.on('init', ({ state, view }) => {
            this.editorView = view
            console.log('editor view: ')
            console.log(this.editorView)
        })  
    },
```

**Comment by hanspagel** [comment]
You can also use the `onInit` configuration [1] when creating the editor. Try this:

```js
new Editor({
  onInit: () => {
    console.log('init')  
  }
})
```

Hope that helps!

[1] https://tiptap.dev/docs/guide/events.html
